neurodocker/templates/matlabmcr.yaml

```diff
@@ -30,16 +30,18 @@ generic:
       LD_LIBRARY_PATH: "$LD_LIBRARY_PATH:/usr/lib/x86_64-linux-gnu:{{ matlabmcr.install_path }}/{{ matlabmcr.mcr_version }}/runtime/glnxa64:{{ matlabmcr.install_path }}/{{ matlabmcr.mcr_version }}/bin/glnxa64:{{ matlabmcr.install_path }}/{{ matlabmcr.mcr_version }}/sys/os/glnxa64:{{ matlabmcr.install_path }}/{{ matlabmcr.mcr_version }}/extern/bin/glnxa64"
       MATLABCMD: "{{ matlabmcr.install_path }}/{{ matlabmcr.mcr_version }}/toolbox/matlab"
     instructions: |
+      export TMPDIR="$(mktemp -d)"
```

Comment:
can't something like that be done in the interfaces/_base.py as well, or you think there is no need really, or it would leak/interfere if exported and then removed?

Reply:
i think doing that would work for singularity but not for docker, because every docker `RUN` command starts up a new shell. so if we export `TMPDIR` in one `RUN` command, the next one won't see that value. that's why i opted to just modify the `yaml` files of programs that do some things in `/tmp`. what do you think?

Reply:
but don't you chain all the commands in the single `RUN` statement via `&&`?  FWIW, for docker it could even be hardcoded `export TMPDIR=${TMPDIR}/neurodocker && ... && rm -rf ${TMPDIR}` although some security-aware folks could raise concerns ;)

Reply:
i'd prefer to keep the `export TMPDIR` bit within the individual template files so that it is obvious where TMPDIR is being set. if it were in `interfaces/_base.py`, things could get confusing in the future once i inevitably forget where TMPDIR comes from :)